src/Makefile.am

```diff
@@ -47,6 +49,7 @@ EXTRA_LIBRARIES = \
   libbitcoin_common.a \
   libbitcoin_server.a \
   libbitcoin_cli.a \
+  libsnark.a \
```

Comment:
It isn't important that they're different AFAIK, but making them the same is fine by me.